101-documentdb-account-create-multi-region-account/azuredeploy.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "databaseAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "The DocumentDB database account name."
+      }
+    },
+    "writeRegionLocation": {
+      "type": "string",
```

Comment:
I've been getting questions about this a lot lately; threw together a wiki page about it that hopefully clears things up: https://github.com/Azure/azure-quickstart-templates/wiki/defaultValue-vs.-azuredeploy.parameters.json-vs.-GEN-*